app/config/frontendAppConfig.scala

```diff
@@ -19,24 +19,38 @@ package config
 import play.api.Play.{configuration, current}
 import uk.gov.hmrc.play.config.ServicesConfig
 
-trait AppConfig {
+trait AppConfigHelpers {
+  def getConfigurationStringOption(key: String) = configuration.getString(key)
+
+  def getConfigurationString(key: String) = configuration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
+
+  def getConfigurationBoolean(key: String) = configuration.getBoolean(key).getOrElse(false)
+}
+
+trait AppConfig extends AppConfigHelpers {
   val assetsPrefix: String
   val analyticsToken: String
   val analyticsHost: String
   val reportAProblemPartialUrl: String
   val reportAProblemNonJSUrl: String
+  val twoStepVerificationHost: String
```

Comment:
val twoStepVerificationHost: String
val twoStepVerificationPath: String
val companyAuthHost: String
val taxAccountRouterHost: String

Those properties are only used in one place, should they be moved to the place are used perhaps?